libraries/networking/src/udt/PacketList.cpp

```diff
@@ -52,6 +52,19 @@ size_t PacketList::getDataSize() const {
     return totalBytes;
 }
 
+size_t PacketList::getMessageSize() const {
+    size_t totalBytes = 0;
+    for (const auto& packet: _packets) {
+        totalBytes += packet->getPayloadSize();
+    }
+    
+    if (_currentPacket) {
+        totalBytes += _currentPacket->getPayloadSize();
+    }
+    
+    return totalBytes;
+}
```

Comment:
Not blocking, but this seems like the kind of think we could keep track of instead of going through the container.